specs/src/specs/networking.md

```diff
@@ -59,9 +59,9 @@ Defined as `MsgWirePayForData`:
 
 Accepting a `MsgWirePayForData` into the mempool requires different logic than other transactions in Celestia, since it leverages the paradigm of block proposers being able to malleate transaction data. Unlike [SignedTransactionDataMsgPayForData](./data_structures.md#signedtransactiondatamsgpayfordata) (the canonical data type that is included in blocks and committed to with a data root in the block header), each `MsgWirePayForData` (the over-the-wire representation of the same) has potentially multiple signatures.
 
-Transaction senders who want to pay for a blob will create a [SignedTransactionDataMsgPayForData](./data_structures.md#signedtransactiondatamsgpayfordata) object, `stx`, filling in the `stx.blobShareCommitment` field [based on the non-interactive default rules](../rationale/data_square_layout.md#non-interactive-default-rules), then signing it to get a [transaction](./data_structures.md#transaction) `tx`.
+Transaction senders who want to pay for a blob will create a [SignedTransactionDataMsgPayForData](./data_structures.md#signedtransactiondatamsgpayfordata) object, `stx`, filling in the `stx.blobShareCommitment` field [based on the non-interaction rules](../specs/data_square_layout.md#non-interaction-rules), then signing it to get a [transaction](./data_structures.md#transaction) `tx`.
 
-Receiving a `MsgWirePayForData` object from the network follows the reverse process: verify using the [non-interactive default rules](../rationale/data_square_layout.md#non-interactive-default-rules) that the signature is valid.
+Receiving a `MsgWirePayForData` object from the network follows the reverse process: verify using the [non-interaction rules](../specs/data_square_layout.md#non-interaction-rules) that the signature is valid.
```

Comment:
[question] if we're renaming non-interactive default rules to non-interaction default rules, should we do a global find + replace of the codebase in this PR?

Reply:
I didn't touch the ADRs as there are an insane number of references, although we might want to in the future

Reply:
I thought we were going to rename it from non-interactive default rules to blob share commitment rules